src/components/Dao/DaoHistoryPage.tsx

```diff
@@ -61,7 +61,7 @@ class DaoHistoryPage extends React.Component<IProps, null> {
                 <tr className={css.proposalHistoryTableHeader}>
                   <th>Proposed by</th>
                   <th>End date</th>
-                  <th>Scheme</th>
+                  <th>Plugin</th>
```

Comment:
plugin? really? that makes no sense to me. Scheme makes more sense. who are we targeting with this language?